src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -37,9 +37,12 @@ public DefaultServer(List<Type> typesToInclude)
             configuration.EnableInstallers();
 
             configuration.DisableFeature<TimeoutManager>();
-            configuration.Recoverability()
-                .Delayed(delayed => delayed.NumberOfRetries(0))
-                .Immediate(immediate => immediate.NumberOfRetries(0));
+            if (!endpointConfiguration.PerformDefaultRetries)
```

Comment:
Looked into the usage and the proposal of @SimonCropp makes sense since we have tests which are testing the default retries. If we didn't have that we would need to define it explicitly and reuse the constants which seems way more cumbersome. I will just remove the boolean from the method